app/src/api/clientApi.ts

```diff
@@ -12,9 +10,7 @@ export interface IncreaseCountRequest {
 
 export interface IncreaseCountResponse {}
 
-export interface ResetRequest {}
-
-export interface ResetResponse {}
+export interface CounterResponse {}
```

Comment:
generic response

Reply:
Leave specific response objects as they belong to different requests.